packages/components/src/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./Spinner.css";
+
+export interface SpinnerProps {
+  readonly small?: boolean;
+  readonly inline?: boolean;
+}
+
+export function Spinner({ small, inline }: SpinnerProps) {
+  const className = classnames(styles.spinner, {
```

Comment:
Maybe we can change the name on this to be a little more specific to the component maybe spinnerStyles ?

Reply:
yup, that would be a better name for sure!! will change this 😄